string actionName
string[] args
string[] preconditions
string[] effects
---
bool effects_met
